data/json/npcs/missiondef.json

```diff
@@ -475,6 +475,18 @@
       "failure": "Quitting already?"
     }
   },
+  {
+    "id": "MISSION_KILL_NEMESIS",
+    "type": "mission_definition",
+    "name": { "str": "Kill nemesis" },
+    "description": "Since the start of the Cataclysm, something has been hunting you relentlessly. You can outrun it but never truly escape it. One of you must die.",
```

Comment:
```suggestion
    "description": "Since the start of the Cataclysm, something has been hunting you relentlessly.  You can outrun it but never truly escape it.  One of you must die.",
```